<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUIColorBarDialog</class>
 <widget class="QWidget" name="GUIColorBarDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Spectrogram Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QCheckBox" name="checkBoxEdit">
     <property name="text">
      <string>Edit Color Bar</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Color Bar</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Out Max color:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButtonOutMax">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>26</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Max value:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="Comp::FITKSciNotationLineEdit" name="lineEditMax"/>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButtonMax">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>26</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Transition number:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="Comp::FITKSciNotationLineEdit" name="lineEditNum"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Min value:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="Comp::FITKSciNotationLineEdit" name="lineEditMin"/>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="pushButtonMin">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>26</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Out Min color:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="pushButtonOutMin">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>26</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Animation</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="Comp::FITKSciNotationLineEdit" name="lineEditInterval"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Time Interval    :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>ms</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_OK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Comp::FITKSciNotationLineEdit</class>
   <extends>QLineEdit</extends>
   <header>FITK_Component/FITKWidget/FITKSciNotationLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
